apmpackage/apm/changelog.yml

```
@@ -1,4 +1,4 @@
-- version: generated
+- version: 8.12.0
```

I think this version we can keep as `generated`
```suggestion
- version: generated
```